### ISSUE TITLE
Container component padding is not applied

### ISSUE THREAD

**Issue by Savinvadim1312** [roninoss/create-expo-stack#378]
## Expected Behavior
The customer `<Container />` component generated by CES is supposed to have padding.

## Possible Solution

```js
import { SafeAreaView } from 'react-native';

export const Container = ({ children }: { children: React.ReactNode }) => {
  return <SafeAreaView className={styles.container}>{children}</SafeAreaView>;
};

const styles = {
  container: 'flex flex-1 p-5',
};
```

The styles have `p-5` however, the `SafeAreaView` ignores the padding property, because it is used internally to manage the safe area. 

One solution would be to render a View with padding inside the SafeAreaView.

**Comment by danstepanov** [comment]
fixed in #b5374ea

**Comment by danstepanov** [comment]
fixed in https://github.com/roninoss/create-expo-stack/commit/b5374ea3cfc9c5ce4e08bd02c7abdedfb0f69282
